fe/fe-core/src/main/java/com/starrocks/qe/ConnectContext.java

```diff
@@ -786,14 +789,15 @@ public boolean enableSSL() throws IOException {
         }
     }
 
-    public StmtExecutor executeSql(String sql) throws Exception {
-        StatementBase sqlStmt = SqlParser.parse(sql, getSessionVariable()).get(0);
-        sqlStmt.setOrigStmt(new OriginStatement(sql, 0));
-        StmtExecutor executor = new StmtExecutor(this, sqlStmt);
-        setExecutor(executor);
-        setThreadLocalInfo();
-        executor.execute();
-        return executor;
+    /**
+     * Release resource of this connection
+     */
+    @Override
+    public void close() {
+        GlobalStateMgr.getCurrentState().getMetadataMgr().removeQueryMetadata();
+        QeProcessorImpl.INSTANCE.unregisterQuery(getExecutionId());
+        remove();
+        LOG.info("close connection {}", connectionId);
```

Comment:
do we really need this log,  in high qps queries, this will lead to bad performance and many logs?